api/v1beta1/ansibletest_types.go

```diff
@@ -205,7 +208,8 @@ type AnsibleTestWorkflowSpec struct {
 	ContainerImage string `json:"containerImage,omitempty"`
 
 	// +operator-sdk:csv:customresourcedefinitions:type=spec
-	// BackoffLimimt allows to define the maximum number of retried executions (defaults to 6).
+	// BackoffLimit allows to define the maximum number of retried executions (defaults to 0).
+	// +kubebuilder:default:=0
```

Comment:
AnsibleTest workflow section should not have default value (the same way as Tobiko and Tempest)
```suggestion
	// +kubebuilder:validation:Optional
```